src/modules/removeData.js

```diff
@@ -0,0 +1,8 @@
+// import saveToLocalStorage from "./saveToDb.js";
+
+// const removeData = (item, dataArray) => {
+//   const filterdData = dataArray.filter((todo, index) => item !== index);
+//   saveToLocalStorage(filterdData);
+// };
+
+// export default removeData;
```

Comment:
Kindly remove the commented-out code to keep the code base clean. It is a bad practice and hence should be avoided.  Since you are not using this file at all, i would recommend deleting this file.